### PR TITLE
Add message journal usage to nodes overview page.

### LINKED ISSUE #1083: Nodes overview page is not showing message journal usage
Was there a reason not to show the message journal usage on the nodes overview page? It was showing the cache utilization in earlier versions to indicate quickly if messages were queuing up or not and I think this was helpful to people.

screen shot 2015-01-26 at 14 03 33

### PR DESCRIPTION
**Pull request by bernd**
Fixes #1083.

Comments?

### CODE REVIEW

**Comment by edmundoa** [comment]
The changes work nicely, although I saw a couple of small things I'm not so convinced about, see comments inline.

File: app/assets/javascripts/periodicals.js [hunk 1]
```diff
@@ -119,6 +119,11 @@ $(document).ready(function() {
                         $(".heap-used", thisHeap).text(numeral(data.used).format("0,0"));
                         $(".heap-total", thisHeap).text(numeral(data.total).format("0,0"));
                         $(".heap-max", thisHeap).text(numeral(data.max).format("0,0"));
+
+                        var journal = $(".journal-uncommitted", thisHeap);
+                        if (journal.length > 0) {
+                            journal.text(numeral(data.journal_uncommitted_entries).format("0,0"));
+                        }
```
Comment (edmundoa) [inline app/assets/javascripts/periodicals.js]: I find confusing mixing heap and journal information under a `updateNodeHeapUsage` method, could you please change that? :)

File: app/views/system/nodes/index.scala.html [hunk 2]
```diff
@@ -159,6 +159,7 @@
                     <div class="bar heap-total-percent" style="width: @(jvm.totalMemoryPercentage()-jvm.usedMemoryPercentage)%"></div>
                 </div>
 
+                <strong><span class="journal-uncommitted">@nodes.get(jvm.getNodeId).getJournalInfo().uncommittedJournalEntries</span> unprocessed messages</strong> in the journal.
```
Comment (edmundoa) [inline app/views/system/nodes/index.scala.html]: Also confusing having an element called `node-heap-usage` with journal information inside. Please change the container name if it is not too hard. I know how those small changes sometimes end up :)

**Comment by bernd** [comment]
I updated the PR according to your comments. Thanks! :smiley: 

One consequence of separating the journal info from the heap info is that each node takes a bit more space now.

4a5c430sadaaaaaelftksuqmcc

What do you guys think? /cc @lennartkoopmann

**Comment by edmundoa** [comment]
Looks good to me, I'll make sure it only uses one line to show all information :+1:

**Comment by bernd** [comment]
Thank you very much! Feel free to merge! :smile:

**Comment by edmundoa** [comment]
Here's the final result :)
screen shot 2015-03-04 at 11 21 28

**Comment by edmundoa** [comment]
Thank you, @bernd! :smile:

**Pull request merged**